app/src/main/java/com/github/polypoly/app/base/menu/lobby/GameParameters.kt

```diff
@@ -35,4 +35,8 @@ data class GameParameters(
         if (roundDuration <= 0 || roundDuration > GameLobbyConstants.RoundDurations.getMaxRoundDuration().toMinutes())
             throw java.lang.IllegalArgumentException("Invalid game duration$roundDuration")
     }
+
+    fun getRoundDurationValue(): GameLobbyConstants.RoundDurations {
+        return GameLobbyConstants.RoundDurations.values().find { it.toMinutes() == roundDuration }!!
```

Comment:
You should use the elvis operator `?:` to set a default still-valid value instead of creating an NPE (operator `!!`) if the value is not within the registered one

Reply:
IMO it helps triggering errors for unwanted values. It was helpfull for finding where we had bad values hardcoded. Now, if we create lobbies as intended using the lobby creator acvitity, we should never have such problems. But I agree, I will switch it to a default.